src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="stars__block">
```

Comment:
An element couldn't be a parent of the block in accordance with BEM. So change the name, you can name it `stars-block` for example, just to avoid misunderstanding